ingestors/documents/pdf.py

```diff
@@ -20,8 +28,13 @@ def ingest(self, file_path, entity):
         """Ingestor implementation."""
         try:
             self.parse_and_ingest(file_path, entity, self.manager)
+        except pikepdf._qpdf.PasswordError as pwe:
+            raise ProcessingException(
+                "Could not extract PDF file. The PDF is protected with a password. Try removing the password protection and re-ingesting the documents."
+            )
         except Exception as ex:
-            raise ProcessingException("Could not extract PDF file: %r" % ex) from ex
+            log.info(f"Failed to ingest PDF: {file_path}. Reason: {ex}.")
+            raise ProcessingException("Could not extract PDF file.")
```

Comment:
Did you intentionally remove the `from ex` from the end of the line here? I think it's generally a good idea to pass the exception up the stack.